datasets/wiki_snippets/wiki_snippets.py

```diff
@@ -179,7 +181,15 @@ def _info(self):
         )
 
     def _split_generators(self, dl_manager):
+        # default configuration downloads Wiki40b
+        if not hasattr(self.config, "wikipedia_name"):
```

Comment:
Not sure why we need this ? The config always has the wikipedia_name attribute no ?

Reply:
There seems to be cases where the Builder is launched with `name='default'` and uses a default `nlp.BuilderConfig`.

Did I miss something / is there a way to ensure that `default` actually uses a `WikiSnippetsConfig`?

Reply:
This should not happen. Do you know in which cases you end up with `name='default'` ?

Reply:
fixed the `BUILDER_CONFIG_CLASS`